### ISSUE TITLE
Brown tumor of bone

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10025]
**Preferred term label:**
Brown tumor of bone

**Synonyms**
Osteoclastoma; osteitis fibrosa cystica

**Definition (free text, please give PubMed ID)**
https://radiopaedia.org/articles/brown-tumour

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
hyperparathyroidism

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000089
- label: Brown tumor
- parents: [Abnormal skeletal morphology (HP:0011842)](https://hpo.jax.org/app/browse/term/HP:0011842)
- pmid: PMID:26354493
- def: A brown tumor is a bone lesion that results from bony resorption by excess osteoclastic activity, and replacement by fibrous tissue and giant cells. Brown tumor is an uncommon pathognomonic sign of hyperparathyroidism.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10025
- synonyms: Osteitis fibrosa cystica(NONE)
